plugins/grid-bookmark/src/GridBookmarkWidget/components/DeleteBookmark.tsx

```diff
@@ -50,16 +52,21 @@ function DeleteBookmarkDialog({
       </DialogTitle>
       <div className={classes.dialogContainer}>
         <Typography>
-          Remove <code>{locString}</code>?
+          Remove row number{' '}
```

Comment:
This should probably be reverted. I.e. `Remove row number {hg19}1:24,341,735..24,607,236` doesn't make sense

Reply:
Fixed :)